reporting-guide.md

```diff
@@ -21,11 +40,17 @@ Discouraged Practices:
 * Driving a web page with a site performance widget by making an API request every time the web page is loaded
 * Requesting a full year of day-level data every day to get a rolling 12-month window
 
-### Time Series Data
-Time Series reports are the simplest report that the API can produce. They simply include information about the performance of a metric (or metrics) over a period of time.
+## Example Time Series Report
 
-Here is a simple 'Page Views' request:
-```json
+The Reports API includes **Time Series** reports. These simple reports include information about the performance of a metric (or metrics) over a period of time.
+
+### Example Request
+
+The following request example includes both a JSON message request body and a `curl` request for the **Page Views** metric.
+
+#### JSON Request Message
+
+```json={line-numbers="yes"}
```

Comment:
Would you double check this in adobe.io when it merges? They don't seem to support all of the bells and whistles that github does.

Reply:
Yes that is definitely the plan. I've talked to various people about this lack of a "preview" problem but that is just the way it is for now--it has to go live to see it in its very last form.